web/e2e-tests/stars.test.ts

```diff
@@ -28,7 +28,7 @@ async function toggle_test_star_message(page: Page): Promise<void> {
 }
 
 async function test_narrow_to_starred_messages(page: Page): Promise<void> {
-    await page.click('#global_filters a[href^="#narrow/is/starred"]');
+    await page.click('#left_sidebar_navigation_list a[href^="#narrow/is/starred"]');
```

Comment:
Let's rewrite `#left_sidebar_navigation_list` to [use hyphens](https://zulip.readthedocs.io/en/latest/subsystems/html-css.html#use-clear-unique-names-for-classes-and-object-ids). Same for any other classes or IDs created in this PR.